### DISCUSSION TITLE
Error Fetching Audio Error in Console

### DISCUSSION THREAD

**Discussion by hanna-daoud** [danny-avila/LibreChat#4118] [Troubleshooting]
### What happened?

Whenever I send a message or I switch between chats, I get the following error message in the console: **[Error fetching audio:]** 

### Steps to Reproduce

1. Open a new chat
2. Send any message
3. Switch to another chat from the history

### Relevant log output

```shell
logger.ts:13 [Error fetching audio:]
```

### Screenshots

[Screencast from 2024-09-18 18-34-06.webm](https://github.com/user-attachments/assets/3ed45c2c-aefb-445b-85ca-1353a58eaaf0)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Disable Text-to-speech or switch the "engine" from settings. You're using a browser where browser/edge TTS voices are not supported.

**Reply by hanna-daoud** [reply]
After switching off Text-to-speech, error is gone
